scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -1614,12 +1628,12 @@ function Uninstall-Services([ref] $RestartNeeded, [bool] $LegacyInstaller) {
     if (Get-Service $EdgeServiceName -ErrorAction SilentlyContinue) {
         Set-Service -StartupType Disabled $EdgeServiceName -ErrorAction SilentlyContinue
         Stop-Service -NoWait -ErrorAction SilentlyContinue -ErrorVariable cmdErr $EdgeServiceName
-        if ($?) {
+        if ($? -and $LASTEXITCODE -eq 0) {
             Start-Sleep -Seconds 7
-            Write-Verbose 'Stopped the IoT Edge service'
+            Write-Verbose "Stopped the IoT Edge service $EdgeServiceName"
         }
         else {
-            Write-Verbose "$cmdErr"
+            Write-Verbose "stopping IoT Edge service $EdgeServiceName failed.  exitcode: $LASTEXITCODE error: $cmdErr"
```

Comment:
Same here, `$LASTEXITCODE` is meaningless for `Stop-Service`

Reply:
fixed per above.